pkg/kubernetes/utils_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/yaml"
 )
 
-var testDataPath = "../../testdata"
+var testDataPath = "../../deployments"
```

Comment:
I think we can keep the testdata directory. we can copy it to deployments instead of removing it. 🙇🏼‍♂️

Reply:
I though that we store all yaml files in deployments. Let's me roll it back

Reply:
Thank you very much.
Let's have two directory! I'm sorry for my lack of communication!

Reply:
Thank you very much!
I updated!

Reply:
Oh, Brave I'm sorry very much. I didn't mean roll it back. I prefer to use `deployments` directory. I mean that, just only for test, we can keep the `testdata` directory too with `deployments`.
I like your change! (we will still use deployments directory! but we need the `testdata` directory for our test.)

Reply:
Thank you Steven!
I understood. Let me update

Reply:
Yes, I'm sorry. In next time, I will tell more details easy to understand. 🙇🏼‍♂️🥲

Reply:
Thank you very much Steven

I also should read the message more carefully. I updated :D